src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -229,9 +228,8 @@ private static string GetUserAgentString()
     {
         try
         {
-            var assemblyVersion = typeof(OtlpExporterOptions).Assembly.GetCustomAttribute<AssemblyInformationalVersionAttribute>();
-            var informationalVersion = assemblyVersion?.InformationalVersion;
-            return string.IsNullOrEmpty(informationalVersion) ? UserAgentProduct : $"{UserAgentProduct}/{informationalVersion}";
+            var assembly = typeof(OtlpExporterOptions).Assembly;
+            return $"{UserAgentProduct}/{assembly.GetPackageVersion()}";
```

Comment:
Minor, remove `UserAgentProduct` and update this line to:

```suggestion
            return $"OTel-OTLP-Exporter-Dotnet/{assembly.GetPackageVersion()}";
```

Reply:
b45be344b341745c895b3eaf0bd61e1303a0cf30